### DISCUSSION TITLE
Can I rename "renovate.json" to ".renovate.json" ?

### DISCUSSION THREAD

**Discussion by bhargu99** [renovatebot/renovate#24990] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

wanted to know whether we can use ".renovate.json"

### What you tried so far.

I changed the name and stopped seeing automatic MR's

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
No
